### PR TITLE
docs: add CSC23 information

### PR DESCRIPTION
**Pull request by strieflin**

### CODE REVIEW

File: docs/community/events/csc23/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+The _Carbyne Stack Conference_ (or _CarbyneStackCon_) is the annual meetup of
+the Carbyne Stack Open Source Community. It is designed to foster collaboration,
+discussion and knowledge sharing within the Carbyne Stack community.
+
+## Registration
+
+CarbyneStackCon '23 (CSC23) is an open event sponsored by Bosch Research taking
+place on November 30th. Everyone interested is welcome to attend! However,
+seats for in-person participation are limited and registration is required
+both for in-person and virtual attendance in order to facilitate our planning.
```
Comment (sbckr) [inline docs/community/events/csc23/index.md:16]: is `both` swapped? -> `... is required for both in-person and...`

File: docs/community/events/csc23/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+!!! important
+    Please use the eventcode **CSC23** to register on the EventXP platform.
+
+<a href="https://event-xp.com" class="md-button md-button--primary">
```
Comment (sbckr) [inline docs/community/events/csc23/index.md:21]: Registration link is https://register.event-xp.com

**Review by sbckr — CHANGES_REQUESTED** [review]

File: docs/community/events/csc23/index.md (hunk 1 above)
Comment (strieflin) [reply docs/community/events/csc23/index.md:16]: Fixed

File: docs/community/events/csc23/index.md (hunk 2 above)
Comment (strieflin) [reply docs/community/events/csc23/index.md:21]: Done

File: docs/community/events/index.md [hunk 3]
```diff
@@ -2,7 +2,12 @@
 
 ## Upcoming Events
 
-N/A
+### [CarbyneStackCon '23][csc23]
+
+![CSC_Logo](/images/events/csc/csc22-logo-no-dates.png){ class="event-logo" }
```
Comment (strieflin) [inline docs/community/events/index.md:7]: Image can be the same as this is the no dates version.

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**